.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: Weekly Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 00:00 on Monday
+    - cron: "0 0 * * 1"
+
+jobs:
+  bump-version-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch all history for all branches and tags
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+
```

Comment:
Maybe this is unneeded as well? We only use JS in this action's code, but I see in the [readme of actions/github-script](https://github.com/actions/github-script#welcome-a-first-time-contributor) that it doesn't set up node. It does use version 7 and not version 6 like in this PR however.

In any case, Node 14 is quite old, the [current version](https://nodejs.org/en/about/previous-releases) is 20.